en/includes/guides/branding/add-custom-layouts.md

```diff
@@ -0,0 +1,185 @@
+
+4. Create the following files and the folder inside the `carbon.super` folder:
+    - `styles.css` file
+    - `script.js` file
+    - `assets` folder
+
+### Step 3: Add a custom layout design
+
+To add a custom layout to the login page:
+
+1. Navigate to `webapps/authenticationendpoint/extensions/layouts/custom/carbon.super/` and follow the instructions provided below:
+
+    1. Copy the code snippet from the [html file](https://github.com/wso2/docs-is/tree/master/en/asgardeo/docs/assets/code-samples/body.html){target="_blank"} into the `body.html` file.
+
+    2. Copy the code snippet from the [css file](https://github.com/wso2/docs-is/tree/master/en/asgardeo/docs/assets/code-samples/style.css){target="_blank"}  into the `styles.css` file.
+
+    3. Copy the code snippet from the [javascript file](https://github.com/wso2/docs-is/tree/master/en/asgardeo/docs/assets/code-samples/script.js){target="_blank"} file  into the `script.js` file.
+
+    4. Copy the [illustration.svg file](https://github.com/wso2/docs-is/tree/master/en/asgardeo/docs/assets/code-samples/illustration.svg){target="_blank"} into the `assets` folder.
+
+    !!! note
+        To update the header and the footer, use the [Branding UI]({{base_path}}/guides/branding/configure-ui-branding/#update-branding).
+
+2. Clone the [identity-apps repository](https://github.com/wso2/identity-apps){target="_blank"} from Github. This directory will be referred to as `IDENTITY-APPS-HOME` in the next step.
+
+2. Build the source code.
+
+    1. Copy the `<IS_HOME>/repository/deployment/server/webapps/authenticationendpoint/extensions/layouts/custom` directory and place it into the `<IDENTITY-APPS-HOME>/identity-apps-core/components/login-portal-layouts/layouts` directory.
+
+    2. Navigate to `<IDENTITY-APPS-HOME>/identity-apps-core/components/login-portal-layouts/layouts` directory and build the source code using `mvn clean install` command (Build should be succeeded for further steps).
+
+    3. Copy `<IDENTITY-APPS-HOME>/identity-apps-core/apps/authentication-portal/src/main/webapp/extensions/layouts/custom` folder (Compiled version of the layout source code) and paste it into the `<IS_HOME>/repository/deployment/server/webapps/authenticationendpoint/extensions/layouts` directory.
+
+4. Refresh the browser and check out the added custom layout.
+
+    !!! tip
+        If the server stops working after the changes are added, restart the server before verifying the changes.
+
+## Best Practices
+
+Use the following best practices when creating a custom layout:
+
+- Add a prefix for the new CSS classes so that the newly added classes will not conflict with existing classes.
+
+- In your development environment, be sure to add the `cache=false` flag as a parameter in the `<layout:main>` tag of all pages that require testing with the custom layout. Remember to remove this flag in the production environment.
```

Comment:
```suggestion
- In your development environment, be sure to add the `cache="false"` flag as a parameter in the `<layout:main>` tag of all pages that require testing with the custom layout. Adding this flag eliminates the need to recompile the layout, as it will be compiled at runtime. Remember to remove this flag in the production environment.
```